R/register_extendr.R

```diff
@@ -28,23 +28,23 @@
 #' @seealso [rextendr::document()]
 #' @export
 register_extendr <- function(path = ".", quiet = FALSE, force = FALSE, compile = NA) {
-  usethis_quiet <- getOption("usethis.quiet")
-  on.exit(options(usethis.quiet = usethis_quiet))
-  options(usethis.quiet = quiet)
+
+  if (quiet) {
+    withr::local_options(list("cli.default_handler" = function(...) { }))
+  }
 
   rextendr_setup(path = path)
 
   pkg_name <- pkg_name(path)
 
-  ui_i("Generating extendr wrapper functions for package: {.pkg {pkg_name}}.")
+  cli::cli_alert_info("Generating extendr wrapper functions for package: {.pkg {pkg_name}}.")
 
   entrypoint_c_file <- rprojroot::find_package_root_file("src", "entrypoint.c", path = path)
   if (!file.exists(entrypoint_c_file)) {
-    ui_throw(
+    cli::cli_abort(c(
       "Unable to register the extendr module.",
-      c(
-        bullet_x("Could not find file {.file src/entrypoint.c }."),
-        bullet_o("Are you sure this package is using extendr Rust code?")
+      "x" = "Could not find file {.file src/entrypoint.c}.",
+      "*" = "Are you sure this package is using extendr Rust code?"
```

Comment:
note that `"*"` creates a colored bullet. If a black bullet is important this can be changed to use a more verbose `cli::cli_fmt()`

Reply:
Note an issue.